spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/mapper/ReflectionColumnMapper.scala

```diff
@@ -30,6 +29,26 @@ abstract class ReflectionColumnMapper[T : ClassTag] extends ColumnMapper[T] {
       paramName: String,
       columns: Map[String, ColumnRef]): Option[ColumnRef]
 
+  protected def annotationForFieldName(
```

Comment:
Not sure why this function def is here and not in the JavaMBean mapper since it is only used there